### ISSUE TITLE
Installed HACS on Hassio 0.98.5.  Not loading, halts Hassio

### ISSUE THREAD

**Issue by martoq** [custom-components/hacs#535]
## Version of HACS
.13.3

## Describe the bug

Fresh install of HACS locks up hassio.  Check config fails.  Reboot doesn't even load HA.

## Debug log

```text

2019-09-16 12:31:54 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant.
2019-09-16 12:33:41 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_protocol.py", line 418, in start
    resp = await task
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_app.py", line 458, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_middlewares.py", line 119, in impl
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/real_ip.py", line 40, in real_ip_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 73, in ban_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 231, in auth_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 128, in handle
    result = await result
  File "/usr/src/homeassistant/homeassistant/components/config/core.py", line 29, in post
    errors = await async_check_ha_config_file(request.app["hass"])
  File "/usr/src/homeassistant/homeassistant/config.py", line 807, in async_check_ha_config_file
    res = await check_config.async_check_ha_config_file(hass)
  File "/usr/src/homeassistant/homeassistant/helpers/check_config.py", line 107, in async_check_ha_config_file
    integration = await async_get_integration_with_requirements(hass, domain)
  File "/usr/src/homeassistant/homeassistant/requirements.py", line 46, in async_get_integration_with_requirements
    await async_process_requirements(hass, integration.domain, integration.requirements)
  File "/usr/src/homeassistant/homeassistant/requirements.py", line 67, in async_process_requirements
    if pkg_util.is_installed(req):
  File "/usr/src/homeassistant/homeassistant/util/package.py", line 45, in is_installed
    return version(req.project_name) in req
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3128, in __contains__
    return self.specifier.contains(item, prereleases=True)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/_vendor/packaging/specifiers.py", line 703, in contains
    item = parse(item)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/_vendor/packaging/version.py", line 31, in parse
    return Version(version)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/_vendor/packaging/version.py", line 200, in __init__
    match = self._regex.search(version)
TypeError: expected string or bytes-like object
```

**Comment by ludeeus** [comment]
There is nothing with this I can do 🤷‍♂ 
Your system have issues installing dependencies if that halts HA you have more issues then the installation of HACS.

Try again later.

**Comment by UksusoFF** [comment]
Same trouble on: 
```
core-ssh:~# hassio ha info
arch: armv7
boot: true
custom: false
image: homeassistant/raspberrypi3-homeassistant
ip_address: 172.30.32.1
last_version: 0.98.5
machine: raspberrypi3
port: 8123
ssl: false
version: 0.97.2
wait_boot: 600
watchdog: true
```

**Comment by UksusoFF** [comment]
@martoq 
Any errors on this page http://hassio.local:8123/developer-tools/info ?

**Comment by ludeeus** [comment]
@martoq ?

**Comment by martoq** [comment]
Apologizes, was out of pocket for work.  I found the problem.  I had Alexa Media package installed under custom components.  I am running 0.99.2 HA.  When booting it would traceback on that package because of the change from MEDIA_PLAYER_SCHEMA to PLATFORM_SCHEMA.  I manually updated media_player.py line 53 to reference PLATFORM_SCHEMA and now HACS loads correctly.  Why HACs would get hung up with HA on this I am uncertain but I figured I would pass along what I found.
